scripts/release_issue_status/main.py

```diff
@@ -135,18 +140,36 @@ def main():
     # rule2: if latest comment is from author, need response asap
     # rule3: if comment num is 0, it is new issue, better to deal with it asap
     # rule4: if delay from latest update is over 7 days, better to deal with it soon.
-    # rule5: if delay from created date is over 30 days and owner never reply, close it.
-    # rule6: if delay from created date is over 15 days and owner never reply, remind owner to handle it.
+    # rule5: if delay from created date is over 30 days, better to close.
+    # rule6: if delay from created date is over 30 days and owner never reply, close it.
+    # rule7: if delay from created date is over 15 days and owner never reply, remind owner to handle it.
     for item in issue_status:
         if item.status == 'release':
             item.bot_advice = 'better to release asap.'
         elif item.author == item.author_latest_comment:
             item.bot_advice = 'new comment for author.'
-        elif item.comment_num == 0:
+        elif item.comment_num == 0 and 'Python' in item.labels:
             item.bot_advice = 'new issue and better to confirm quickly.'
+            if 'auto-link' not in item.labels:
+                try:
+                    update_issue_body(sdk_repo, rest_repo, item.link)
+                except Exception as e:
+                    item.bot_advice = 'failed to modify the body of the new issue. Please modify manually'
+                    item.labels.append('attention')
+                item.labels.append('auto-link')
+                item.issue_object.set_labels(*item.labels)
+            try:
+                time.sleep(5)
+                reply = rg.ReplyGenerator(issue_object=item.issue_object, rest_repo=rest_repo)
+                reply.run()
```

Comment:
no need to do with OO style. Just define a function to do it like `update_issue_body`